alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java

```diff
@@ -0,0 +1,126 @@
+//https://vehiculos.doncomos.com/calcular-consumo-gasolina
+
+class CarBehavior {
+    static class Car {
+        String name;
```

Comment:
I will recommend to just to have a single class called Car, in either case please define your Fields with private access modifier and create access methods is necessary (getters and setters).

Reply:
Update: Fields defined  with private access modifier and access methods (getters) created in the car project.